oidc_provider/views.py

```diff
@@ -174,6 +178,23 @@ def post(self, request, *args, **kwargs):
 
             return redirect(uri)
 
+    @staticmethod
+    def strip_prompt_login(path):
+        """
+        Strips 'login' from the 'prompt' query parameter.
+        """
+        uri = urlsplit(path)
+        query_params = parse_qs(uri.query)
+        prompt = query_params['prompt']
+        if 'login' in prompt:
+            prompt.remove('login')
+        if prompt:
+            query_params['prompt'] = prompt
```

Comment:
I was trying to explain that you don't need this `query_params['prompt'] = prompt` in a fragment:
```
    if prompt:
        query_params['prompt'] = prompt
    else:
        del query_params['prompt']
```

I can agree that you want to remove `prompt` parameter when it is empty so what about this?:
```
    if not prompt:
        del query_params['prompt']
```